build/flaws.js

```diff
@@ -497,10 +499,25 @@ async function fixFixableFlaws(doc, options, document) {
           timeout: 10000,
           retry: 3,
         });
+        const fileType = await FileType.fromBuffer(imageBuffer);
+        if (!fileType) {
+          throw new Error(
```

Comment:
Maybe i'm using it in a weird way, but thinking towards when this might be run across the entire repo, but should this be a `console.warn` and `continue`

Reply:
It's too late at that point in my opinion. The UI says there are fixable flaws and click to attempt that. If that doesn't work, it's best elevate the problem rather than "sneaking away". It would probably different if we had a good UI for returning to the user "You told me to try to fix all fixable flaws, but I could only do 7 out of 10 flaws because of the following errors..."
But we don't have that. At least not now.